packages/content-source-maps/README.md

```diff
@@ -1,3 +1,95 @@
 # Content Source Maps
 
-> The base for generating Content Source Maps with Contentful
+Automatically tag visual fields such as short text, rich text and assets to streamline the setup process for Live Preview Inspector Mode, eliminating the time required for manually inserting data attributes to activate Inspector Mode.
```

Comment:
What do you think about
`Automatically add inspector mode to visual fields such as ...